### PR TITLE
Add canonical links /vm 

### PR DESCRIPTION
**Pull request by LucasSaintarbor**
Adding canonical links to /vm section pages in Harvester.

Tied to #341

### CODE REVIEW

File: versioned_docs/version-v0.3/vm/access-to-the-vm.md [hunk 1]
```diff
@@ -11,6 +11,10 @@ keywords:
 Description: Once the VM is up and running, it can be accessed using either VNC or the serial console from the Harvester UI.
 ---
 
+<head>
+  <link rel="canonical" href="https://docs.harvesterhci.io/v1.1versioned_/version-v0.3/vm/access-to-the-vm"/>
```
Comment (vickyhella) [inline versioned_docs/version-v0.3/vm/access-to-the-vm.md:15]:
```suggestion
  <link rel="canonical" href="https://docs.harvesterhci.io/v1.1/vm/access-to-the-vm"/>
```

File: versioned_docs/version-v0.3/vm/backup-restore.md [hunk 2]
```diff
@@ -11,6 +11,10 @@ keywords:
 Description: VM backups are created from the Virtual Machines page. The VM backup volumes will be stored in the Backup Target(an NFS or S3 server) and they can be used to either restore a new VM or replace an existing VM.
 ---
 
+<head>
+  <link rel="canonical" href="https://docs.harvesterhci.io/v1.1versioned_/version-v0.3/vm/backup-restore"/>
```
Comment (vickyhella) [inline versioned_docs/version-v0.3/vm/backup-restore.md:15]: ditto and all the links below need to be modified accordingly~

Comment (vickyhella) [inline versioned_docs/version-v0.3/vm/backup-restore.md:15]:
```suggestion
  <link rel="canonical" href="https://docs.harvesterhci.io/v1.1/vm/backup-restore"/>
```

**Review by vickyhella — CHANGES_REQUESTED** [review]

File: versioned_docs/version-v0.3/vm/backup-restore.md (hunk 2 above)
Comment (vickyhella) [reply versioned_docs/version-v0.3/vm/backup-restore.md:15]: ditto and all the links below need to be modified accordingly~

**Comment by LucasSaintarbor** [comment]
Opening new PR with updated script

**Pull request closed without merging**